go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -81,13 +81,23 @@ func (agent *ActionAgent) restoreDataLocked(ctx context.Context, logger logutil.
 	// Record local metadata values based on the original type.
 	localMetadata := agent.getLocalMetadataValues(originalType)
 	tablet := agent.Tablet()
-	dir := fmt.Sprintf("%v/%v", tablet.Keyspace, tablet.Shard)
 
+	keyspaceDir := tablet.Keyspace
+	keyspaceName := tablet.Keyspace
```

Comment:
Does this need to be stored in a new var? Why not just refer to `tablet.Keyspace` directly?